tests/performance/utils.js

```diff
@@ -21,13 +21,67 @@ function median( array ) {
  * @return {string} Result file name.
  */
 function getResultsFilename( fileName ) {
-	const prefixArg = process.argv.find( ( arg ) => arg.startsWith( '--prefix' ) );
-	const fileNamePrefix = prefixArg ? `${prefixArg.split( '=' )[1]}-` : '';
+	const prefixArg = process.argv.find( ( arg ) =>
+		arg.startsWith( '--prefix' )
+	);
+	const fileNamePrefix = prefixArg ? `${ prefixArg.split( '=' )[ 1 ] }-` : '';
 	const resultsFilename = fileNamePrefix + fileName + '.results.json';
 	return resultsFilename;
 }
 
+/**
+ * Returns time to first byte (TTFB) using the Navigation Timing API.
+ *
+ * @see https://web.dev/ttfb/#measure-ttfb-in-javascript
+ *
+ * @return {Promise<number>}
+ */
+async function getTimeToFirstByte() {
+	return page.evaluate(
+		() =>
+			new Promise( ( resolve ) => {
+				new PerformanceObserver( ( entryList ) => {
+					const [ pageNav ] =
+						entryList.getEntriesByType( 'navigation' );
+
+					resolve( pageNav.responseStart );
```

Comment:
This is reporting TTFB as `responseStart` without accounting for `startTime`. Shouldn't we subtract `startTime` to account for situations where its value is different from zero?

[This](https://github.com/WordPress/gutenberg/blob/034f49c138ed82c75109df3a37af66c41913b18b/packages/e2e-tests/specs/performance/front-end-block-theme.test.js#L68) is how we do it in Gutenberg.